Cargo.toml

```diff
@@ -53,3 +53,7 @@ rand = { version = "0.9", default-features = false }
 thiserror = { version = "2.0", default-features = false }
 vm-core = { package = "miden-core", version = "0.13", default-features = false }
 vm-processor = { package = "miden-processor", version = "0.13", default-features = false }
+
+# TODO: remove when pingora releases a fix https://github.com/cloudflare/pingora/issues/568
+[patch.crates-io]
+sfv = { git = "https://github.com/undef1nd/sfv.git", tag = "v0.9.4" }
```

Comment:
Is there a way to refer to a crates.io version that works? We can't publish crates with git dependencies.

Reply:
I'm looking for another workaround. Apparently, patching is only supported using different sources so we cannot use another crates.io version.

Reply:
I couldn't find a way to do that. Maybe forking and fixing pingora and publishing it but does not sound like a good idea. Maybe the best, until a proper fix by them is in place, is to add a caution to the proving service `README` with a message of installing only with `--locked`.